<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adduser</class>
 <widget class="QMainWindow" name="adduser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>302</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QRadioButton" name="btnAD">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>200</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Quản lý</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblNUser">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>                    THÊM NGƯỜI DÙNG MỚI</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>177</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cấp quyền:</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="btnKH">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>200</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Khách hàng</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnHuy">
    <property name="geometry">
     <rect>
      <x>271</x>
      <y>250</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Huỷ</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/res/res/icon/discard.png</normaloff>:/res/res/icon/discard.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLuu">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>250</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Lưu</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/res/res/icon/save.png</normaloff>:/res/res/icon/save.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnShow">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>130</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/res/res/icon/showpw.png</normaloff>:/res/res/icon/showpw.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>89</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="lblNID">
     <property name="text">
      <string>Tên người dùng</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leditNID">
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>15</number>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="lblNPW">
     <property name="text">
      <string>Mật khẩu          </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leditNPW">
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>15</number>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
